sdk/cognitiveservices/textanalytics/Azure.AI.TextAnalytics/tests/samples/Sample5_ExtractKeyPhrases.cs

```diff
@@ -24,7 +24,8 @@ public void ExtractKeyPhrases()
             string input = "My cat might need to see a veterinarian.";
 
             Debug.WriteLine($"Extracting key phrases for input: \"{input}\"");
-            var keyPhrases = client.ExtractKeyPhrases(input).Value;
+            var result = client.ExtractKeyPhrases(input).Value;
```

Comment:
Should these locals be declared explicitly? Not sure what they return at a glance. IIRC, only the result of instantiation can be assigned to `var`.